franklin/builder/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+import logging
+from rest_framework.permissions import AllowAny
+from rest_framework import status
+from rest_framework.response import Response
 
+from rest_framework import generics
+from .models import Environment
 # Create your views here.
+
+from .serializers import EnvironmentStatusSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class UpdateBuildStatus(generics.UpdateAPIView):
+    queryset = Environment.objects.all()
+    serializer_class = EnvironmentStatusSerializer
+    permission_classes = (AllowAny,)
```

Comment:
I assume we are just skipping security on this endpoint for the moment and will eventually create special access that only allows builder to call this.